### ISSUE TITLE
Connected peers exceeded max count

### ISSUE THREAD

**Issue by codeman-crypto** [Beldex-Coin/beldex#74]
The max peers(in and out) connected to a node should be 40 but, the nodes are connect to more than  40 peers.

<img width="1124" alt="Screenshot 2022-06-09 at 3 41 05 PM" src="https://user-images.githubusercontent.com/44291130/172823317-689f1925-4ed2-41b8-9ca2-7fb808167477.png">

The above screenshot shows 36 `out` and 32 `in` so, totally there are 69 peers connected which is higher than the max peer connection count

**logs**
```
2022-05-31 05:34:33.987	[P2P7]	ERROR	net	contrib/epee/include/epee/net/levin_protocol_handler_async.h:342	[15.207.44.251:19090 OUT] [levin_protocol] -->> start_outer_call failed
2022-05-31 05:34:33.987	[P2P7]	ERROR	net	contrib/epee/include/epee/net/levin_protocol_handler_async.h:342	[49.205.84.124:31636 INC] [levin_protocol] -->> start_outer_call failed 

```

**Note**: Looks like the dead connections are not being dropped.

**Comment by victor-tucci** [comment]
In the function try_ping  `        LOG_PRINT_CC_L0(context,"ping handler before callback" );
        cb();
        //zone.m_net_server.get_config_object().close(ping_context.m_connection_id);
`  in [net_node.inl](https://github.com/Beldex-Coin/beldex/blob/dev/src/p2p/net_node.inl#L2126)
this may can't drop the connection id.
i enable this close() function and looking into the logs.

**Comment by victor-tucci** [comment]
checked with the close(). the timeout peers handshake are closed.  #75

**Comment by codeman-crypto** [comment]
Resolved in #75
